### ISSUE TITLE
BRJS todo app not loading in IE8

### ISSUE THREAD

**Issue by thanhc** [BladeRunnerJS/brjs#1354]
The following error is being thrown in IE8 when opening the todo app:

Unexpected call to method or property access, when calling the following line in app.js line 46:

todoAppEl.appendChild(m_eElement)

both todoAppEl and m_eElement are objects.

**Comment by ioanalianabalas** [comment]
Fixed by #1385, plus a minor change to the brjstodo app.

**Comment by andyberry88** [comment]
@thanhc can you add a comment explaining why this was moved back to dev?

**Comment by ioanalianabalas** [comment]
@andyberry88, @thanhc told me about it and he is encountering this issue during testing (it's in the PR #1385):

This fix is causing any new app to be created to display blade workbench components in the app itself.

Steps:
create a new app and add a blade then launch app

**Comment by ioanalianabalas** [comment]
I have left the following comment on the corresponding PR but thought I should share it here as well:

I have tested the scenario mentioned by @thanhc but it seems like this behaviour was not introduced by this branch. I have tried creating a new app on both this branch and the `develop` branch, and I am still seeing the blade in the app itself. @thanhc will be raising a separate issue for that.
